rwengine/src/ai/CharacterController.cpp

```diff
@@ -239,6 +239,28 @@ bool Activities::EnterVehicle::update(CharacterObject *character,
         } else if (character->animator->getAnimation(AnimIndexAction) ==
                    anm_enter) {
             if (character->animator->isCompleted(AnimIndexAction)) {
```

Comment:
Note that animations do not necessarily describe what goes on in the game world.
The player might be warped into the car if the doors are not accessible. Similarily the animation might not play if the script puts the player in a vehicle (will rewards still work?!) etc.

I feel like we should instead have some kind of callback which signals that a player has entered a vehicle (possibly informing about seatIndex, vehicle, character (not necessarily as we can probably retrieve it from the seat)).

However, the current implementation will probably work too.